src/circuit/gadget/ecc/chip/add_incomplete.rs

```diff
@@ -136,8 +136,8 @@ impl Config {
         )?;
 
         let result = NonIdentityEccPoint {
-            x: CellValue::<pallas::Base>::new(x_r_var, x_r),
-            y: CellValue::<pallas::Base>::new(y_r_var, y_r),
+            x: x_r_var,
+            y: y_r_var,
```

Comment:
Consider renaming these to `x_r_cell` and `y_r_cell` for consistency.

Reply:
I was trying to keep the diff as small as possible 😅 but yes, I can go through and clean up variable names.

Reply:
Decided to not do this in this PR, it's already large enough. This can be done post-migration to `halo2_gadgets`.